flag.go

```diff
@@ -9,21 +9,23 @@ import (
 )
 
 type Flag struct {
-	Default     interface{}
-	Description string
-	Name        string
-	Short       string
-	Value       interface{}
+	Default         interface{}
+	Description     string
+	Name            string
+	Short           string
+	Value           interface{}
+	skipHelpCommand bool
 
 	kind string
 }
 
-func BoolFlag(name, short, description string) Flag {
```

Comment:
keep it as it is, since probably this one used serveral places, it will break that
If you need create another method with name `BoolFlagWithSkipHelp`